### PR TITLE
feat(isort) Add isort pre commit hook

### PR DESCRIPTION
**Pull request by evanh**
This will require running `make setup-git` in order to get the right version of `pre-commit`.

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[settings]
+profile=black
+src_paths=snuba
```
Comment (lynnagara) [inline .isort.cfg:3]: Do we not want to format the tests as well?

File: .isort.cfg [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+multi_line_output=3
+line_length=88
```
Comment (lynnagara) [inline .isort.cfg:5]: Why 88?

File: .isort.cfg [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+atomic=true
+include_trailing_comma=true
```
Comment (lynnagara) [inline .isort.cfg:7]: Does profile=black not already imply most of these other settings?

File: .isort.cfg (hunk 2 above)
Comment (evanh) [reply .isort.cfg:5]: That's black's limit as well. Otherwise the two will conflict.

File: .isort.cfg (hunk 3 above)
Comment (evanh) [reply .isort.cfg]: Yep, good point. I removed the unnecessary configs.

**Review by lynnagara — APPROVED** [review]

**Pull request merged**